### DISCUSSION TITLE
How can we retrieve the turbulence data?

### DISCUSSION THREAD

**Discussion by ZilD117** [weather-gov/api#700] [General]
Hey guys :) Hope y'all are doing well! I'm a college student trying to make a project on presenting the turbulence forecast for any given flight path. My thought on this is to retrieve the trieve the turbulence level of the grids within the path and make it a beautiful plot. I've spent more than 10 hours on finding the appropriate data to back up my idea but I was not able to find where AWC/NOAA provide the raw data. I did find the turbulence plot from GFA tho (https://aviationweather.gov/graphics/). Is there any chance someone happen to know where these type of data can be found?

**Comment by sullynole** [comment]
Might try AWC's API here: https://aviationweather.gov/data/api/#/

**Reply by ZilD117** [reply]
Thanks for sharing that! AWC's api does have some of the turbulence data included in gairmet but I found the data seemed incomplete (like only a few grids has the Turbulence Severity label). I was looking for the data source in which each of the grids has the turbulence severity data included. thanks for the help tho!

**Reply by seth-wood** [reply]
@ZilD117 Did you ever find a solution? I'm also looking for this too.
